File: ALGORITMOS-E-ESTRUTURAS-DE-DADOS-II (PT2)/#Teorico/0 - Revisao/3 - Entrada e Saída/inputOutput.java
import java.util.Scanner; //Tem que importar a iblioteca que for utilizada

public class inputOutput {
    public static void main(String[] args) {
        System.out.print(""); //Saída Padrão
        System.out.println(""); //Saída Padrão + Pula 1 linha

        //Deve-se instanciar um objeto, no caso teclado
        Scanner teclado = new Scanner(System.in);
            String s = teclado.nextLine();
            int num = teclado.nextInt();
            float num2 = teclado.nextFloat();
        teclado.close();

        //Exemplo
        String nome;
        int idade;
        float altura;

        Scanner entrada = new Scanner(System.in);
            System.out.print("Digite seu primeiro nome: ");
            nome = entrada.nextLine();

            System.out.print("Digite sua idade: ");
            idade = entrada.nextInt();

            System.out.print("Digite sua altura: ");
            altura = entrada.nextFloat();

            System.out.print("\n" + nome + " tem " + idade + " anoes de idade e mede " + altura + "m.");
        entrada.close();

    }
}

File: ALGORITMOS-E-ESTRUTURAS-DE-DADOS-II (PT2)/#Teorico/0 - Revisao/3 - Entrada e Saída/inputOutput.cpp
#include <iostream>

using namespace std;

int main(int argc, char const *argv[]) {
    int x;
    std::cin >> x; //Entrada

    std::cout << "Hello World!\n"; //Saída

    //Exemplo
    char nome[20];
    int idade;
    float altura;

    cout << "Digite seu primeiro nome: ";
    cin >> nome;

    cout >> "Digite sua idade: ";
    cin >> idade;

    cout << "Digite sua altura: ";
    cin >> altura;

    cout << nome << " tem " << idade << " anos e mede" << altura "m.";
    return 0;
}